source/frontend/src/models/generate/CompensationRequisition/GenerateCompReqFileLease.ts

```diff
@@ -0,0 +1,55 @@
+import { ApiGen_CodeTypes_LeaseStakeholderTypes } from '@/models/api/generated/ApiGen_CodeTypes_LeaseStakeholderTypes';
+import { ApiGen_CodeTypes_LessorTypes } from '@/models/api/generated/ApiGen_CodeTypes_LessorTypes';
+import { ApiGen_Concepts_Lease } from '@/models/api/generated/ApiGen_Concepts_Lease';
+import { ApiGen_Concepts_LeaseStakeholder } from '@/models/api/generated/ApiGen_Concepts_LeaseStakeholder';
+import { ApiGen_Concepts_PropertyLease } from '@/models/api/generated/ApiGen_Concepts_PropertyLease';
+
+import { Api_GenerateH120Property } from '../acquisition/GenerateH120Property';
+import { Api_GenerateOwner } from '../GenerateOwner';
+import { Api_GenerateProduct } from '../GenerateProduct';
+import { Api_GenerateProject } from '../GenerateProject';
+import { ICompensationRequisitionFile } from './ICompensationRequisitionFile';
+
+export class GenerateCompReqFileLease implements ICompensationRequisitionFile {
```

Comment:
Please rename this to "Api_Generate..." to be consistent with naming convention of generate classes

Reply:
Updated